tests/baselines/reference/privateNameInInExpression.errors.txt

```diff
@@ -46,7 +43,7 @@ tests/cases/conformance/classes/members/privateNames/privateNameInInExpression.t
     
             for (#field in v) { /**/ } // Bad - 'in' not allowed
                  ~~~~~~
-!!! error TS1451: Private identifiers are only allowed in class bodies and may only be used as part of a class member declaration, property access, or on the left-hand-side of an 'in' expression
+!!! error TS2304: Cannot find name '#field'.

```

Comment:
this error should also be removed

Reply:
This is weird, the baseline is different from what I run with `node lib/local/tsc.js`

Reply:
With tsc.js

![image](https://user-images.githubusercontent.com/52666982/146288609-398e1697-acce-466c-b393-826e22e4c7b6.png)